api_fhir_r4/apps.py

```diff
@@ -41,11 +42,11 @@
         "fhir_code_for_widowed": "W",
         "fhir_code_for_unknown": "U"
     },
-    "R4_fhir_location_role_type": {
+    "R4_fhir_location_site_type": {
         "system": "http://hl7.org/fhir/v3/ServiceDeliveryLocationRoleType/vs.html",
-        "fhir_code_for_hospital": "H",
-        "fhir_code_for_dispensary": "D",
-        "fhir_code_for_health_center": "C"
+        "fhir_code_for_hospital": "HOSP",
+        "fhir_code_for_dispensary": "COMM",
+        "fhir_code_for_health_center": "OP",
```

Comment:
Where did you get this code? Extension?

Reply:
From the FHIR url mentionned above ....

Reply:
I can find HOSP and COMM but OP is not in the list. Should we look for another code?

Reply:
you are right it is a type, I meant OF
http://hl7.org/fhir/v3/RoleCode/cs.html#v3-RoleCode-OF

Reply:
I changed it to OF